langchain4j-weaviate/src/main/java/dev/langchain4j/store/embedding/weaviate/WeaviateEmbeddingStore.java

```diff
@@ -70,6 +70,7 @@ public class WeaviateEmbeddingStore implements EmbeddingStore<TextSegment> {
      * @param securedGrpc       The GRPC connection is secured
      * @param grpcPort          The port, e.g. 50051. This parameter is optional.
      * @param textFieldName     The name of the field that contains the text of a {@link TextSegment}. Default is "text".
+     * @param metadataFieldName metadataFieldName The name of the field where {@link Metadata} entries are stored. Default is "_metadata". If set to empty string, {@link Metadata} entries will be stored in the root of the Weaviate object.
```

Comment:
Could you please add a bit of details in `docs/docs/integrations/embedding-stores/weaviate.md`?

Reply:
So I looked at the doc and I am not sure what you want me to put there exactly. To documentation is pretty barebones as is for all the integrations so not sure what you would want that to look like

Reply:
Ok add a table based on the comments

Reply:
@Jaland thanks a lot!